nautobot/dcim/filters/__init__.py

```diff
@@ -1881,6 +1888,12 @@ class ControllerManagedDeviceGroupFilterSet(NautobotFilterSet):
             "name": "icontains",
         }
     )
+    capabilities = django_filters.MultipleChoiceFilter(
+        choices=ControllerCapabilitiesChoices,
+        null_value=None,
+        lookup_expr="icontains",
```

Comment:
I believe MultipleChoiceFilters get the automatically created `__ic` filter. If so we should remove the `lookup_expr` from the default filter?

Reply:
They are automatically created and don't work as MultipleChoiceFilter doesn't support Arrays. Out of all the automatically created filters, the only one that worked was the `icontains`.

Reply:
If all of the automatically created filters are broken we either need to fix or remove them